pkg/error/aws_list_error_handler.go

```diff
@@ -0,0 +1,35 @@
+package error
```

Comment:
Yep I didn't really know what to do with it. I could also just move this to scanner as for know it's the only user of this handler.

Reply:
Yeah I think this could be a good move. We could image have a collection of error handlers of multiples kind in the future. It can make sense to create a dedicated pkg under `remote/` WDYT ?

Reply:
Actually we do not need any AWS specific error handler imho. Enumeration error can occur from any supplier, I think we should have one error handler to handle every enum error coming from any provider.

Reply:
What about if we need them in our middlewares ?

Reply:
That's another topic for me, we should make theses error handler only related to remote scan operations. 
Our actual middleware are not meant to talk with remote, their goal is only to work on resourceFromState and resourceFromRemote slices.